docs/manual/database/live-query.md

```diff
@@ -132,13 +142,161 @@ func main() {
 **docs:** An array of latest result set.
 **type:** Type of operation due to which the `onSnapshot` is called. It can have one of the following values:
 - **initial** - Called only once for the initial data on successful subscription
-- **write** - Whenever any data is added or updated
+- **insert** - Whenever any data is added
+- **update** - Whenever any data is updated
 - **delete** - Whenever any data is deleted
+**changedDoc:** The doc that changed.
 
 `onError` function is called with the `error` if there was any error subscribing to data.
 
 As you would have noticed the `subscribe` function returns an `unsubscribe` function. You should call this function whenever you want to unsubscribe to the changes.
 
+## Setting the liveQuery options:
+You can set the liveQuery options using the `options()` function.  
+The function helps to set `changesOnly` to true or false(default).  
+If `changesOnly` is false, it caches the docs. `onSnapshot` will be called with all 3 parameters set.  
+If `changesOnly` is true, it does not cache the docs and also ignores the initial values. `onSnapshot` will be called with only the last 2 parameters set.  
+
+Here's a code snippet to do this:
+
+ <div class="row tabs-wrapper">
+  <div class="col s12" style="padding:0">
+    <ul class="tabs">
+      <li class="tab col s2"><a class="active" href="#live-query-options-js">Javascript</a></li>
+      <li class="tab col s2"><a href="#live-query-options-java">Java</a></li>
+      <li class="tab col s2"><a href="#live-query-options-python">Python</a></li>
+      <li class="tab col s2"><a href="#live-query-options-golang">Golang</a></li>
+    </ul>
+  </div>
+  <div id="live-query-options-js" class="col s12" style="padding:0">
+    <pre>
+      <code>
+import { API, cond, or, and } from 'space-api';
```

Comment:
In the examples we just need to have the code concerning the live query. The imports and error handling isnt required

Reply:
We have nearly always included whole code snippets, including imorts and error handling

Reply:
Have a look at read data js docs.